api/storage_service.cc

```diff
@@ -495,7 +496,22 @@ void set_storage_service(http_context& ctx, routes& r) {
 
     ss::remove_node.set(r, [](std::unique_ptr<request> req) {
         auto host_id = req->get_query_param("host_id");
-        return service::get_local_storage_service().removenode(host_id).then([] {
+        std::vector<sstring> ignore_nodes = split(req->get_query_param("ignore_nodes"), ",");
```

Comment:
inet_address?

Reply:
It is converted to inet_address below. We can only get string from the query parameter.

Reply:
Well, at least we can start with an array.